doc/base/classes.xml

```diff
@@ -53703,15 +53718,19 @@
 			<argument index="0" name="v" type="var">
 			</argument>
 			<description>
+				Inverse-transforms the given vector "v" by this transform.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="origin" type="Vector2" setter="" getter="" brief="">
+			The origin of the transform. Which is the translation offset.
```

Comment:
Maybe simply
> The translation offset of the transform
since the name origin already means that we are talking about the Transform2D origin (same in Transform)

Reply:
Thx :) :+1: